### ISSUE TITLE
❓ Question: Share a file that already exist in pingvin host machine

### ISSUE THREAD

**Issue by Jaeger87** [stonith404/pingvin-share#476]
### 🙋‍♂️ Question

Hi, i didn't find how to share a file without upload in pingvin by web page. Is possible right now or it could be a feature for the future?

**Comment by stonith404** [comment]
No, this is currently not possible. It's probably not that easy to implement as all the uploaded files are indexed inside the database. This would probably require something like a background job that checks if new files were added.

Nevertheless, feel free to [create a feature request](https://github.com/stonith404/pingvin-share/issues/new?assignees=&labels=feature&projects=&template=feature.yml&title=%F0%9F%9A%80+Feature:+) :)
